app/api/client_connectors.py

```diff
@@ -53,8 +55,23 @@ def get_graphql_fragment():
 MUT_CREATE_CONNECTOR = gql(
     _CONNECTOR_FRAGMENT
     + """
-mutation CreateConnector($name: String, $remoteNetworkId: ID!) {
-  connectorCreate(name: $name, remoteNetworkId: $remoteNetworkId) {
+mutation CreateConnector($name: String, $hasStatusNotificationsEnabled: Boolean!, $remoteNetworkId: ID!) {
```

Comment:
I would also add this `has_status_notifications_enabled` attribute to the [`ConnectorFactory`](https://github.com/Twingate/kubernetes-operator/blob/7a135f26d2e4cb6bcbdf4839ac76f62149443f1f/app/api/tests/factories.py#L64) 
so that the [`test_connector_create()`](https://github.com/Twingate/kubernetes-operator/blob/065c14687389e0de66575709d5b05e6403f55e39/app/api/tests/test_connectors.py#L59) would test this new argument.